packages/database/migration/20260812110000_eng_2343_better_auth_17_resource_model/migration.sql

```
@@ -0,0 +1,173 @@
+CREATE TABLE "oauthClientAssertion" (
+    "id" TEXT NOT NULL,
+    "expiresAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "oauthClientAssertion_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "oauthResource_identifier_key" ON "oauthResource"("identifier");
+
+-- CreateIndex
+CREATE INDEX "oauthClientResource_clientId_idx" ON "oauthClientResource"("clientId");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)